shared/macros.jinja

```diff
@@ -540,3 +540,17 @@ ocil_clause: "the correct value is not returned"
     {{%- endif %}}
 {{%- endmacro %}}
 
+
+{{# Call e.g. {{{ if_product_name_is(rhel6="Old RHEL", rhel5="Very old RHEL") }}} #}}
+{{% macro if_product_name_is() -%}}
```

Comment:
why invent a new code flow mechanism when jinja2 already has if? This does't bring any benefits over doing a simple `if`...

Reply:
This design is simple, it does not abuse jinja in any way, and it allows the macro to remain the same if we add/rename a product.

Reply:
my comment is not about the actual implementation of the macro, that's fine. my comment is about why you would even define a macro for this when you can use language features that are much easier to read for everyone

Reply:
OK, I see your point now. The macro is less verbose, but still readable. However, as the functionality is not used in many files, I agree with you as I give it a second thought.